flavors/README.md

```diff
@@ -35,6 +35,12 @@ We provide three flavors for different Exasol versions. The flavors only specify
   - Available languages: Java 11, Python 2.7, Python 3.6 and R 3.4
   - [Here](standard-EXASOL-6.2.0/FLAVOR_DESCRIPTION.md) you can find more details about the flavor, such as the provided packages.
 
+- python-3.6-minimal-6.2.0
+  - Base Image: ubuntu:18.04
+  - Compatible Exasol Versions: 7.0.\*, 6.2.\*
+  - Available languages: Python 3.6
+  - [Here](https://github.com/exasol/script-languages/tree/master/flavors/python-3.6-minimal-EXASOL-6.2.0/FLAVOR_DESCRIPTION.md) you can find more details about the flavor, such as the provided packages.
```

Comment:
And a link in the [flavor description](https://github.com/exasol/script-languages/tree/master/flavors/python-3.6-minimal-EXASOL-6.2.0/FLAVOR_DESCRIPTION.md) of python-3.6 minimal is also broken. The link to 

https://github.com/exasol/script-languages/blob/master/flavors/python-3.6-minimal-EXASOL-6.2.0/flavor_base/flavor_base_deps/packages/pip3_packages

needs to be

https://github.com/exasol/script-languages/blob/master/flavors/python-3.6-minimal-EXASOL-6.2.0/flavor_base/flavor_base_deps/packages/python3_pip_packages

We really need the link checker :sweat_smile:

Reply:
the submodule needs to be pushed and merged and then you need to update the submodule link to the new commit in master of script-languages

Reply:
I created https://github.com/exasol/script-languages-release/issues/264